# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: startupcache/PStartupCache.ipdl
# Commit: 6450088b6b73
# Full Hash: 6450088b6b73ffc17c79cd7097b1bfe30d00e207
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2020-09-14 21:29:41
# Regressor Bug: 1627075
# File Overlap Count: 1
# Description:
#   Bug 1656261 - Back out all recent StartupCache work r=RyanVM
#   
#   This backs out all work from bug 1627075 as well as all of its
#   descendents. There were a few conflicts when backing this out but
#   overall it was pretty clean, so I would say it's a fairly mild
# ==============================================================================

diff -r b0fabcd60430 -r 6450088b6b73 startupcache/PStartupCache.ipdl
--- a/startupcache/PStartupCache.ipdl	Mon Sep 14 17:32:40 2020 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,31 +0,0 @@
-/* -*- Mode: C++; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 8 -*- */
-/* vim: set sw=4 ts=8 et tw=80 ft=cpp : */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-include protocol PContent;
-
-using class mozilla::TimeStamp from "mozilla/TimeStamp.h";
-using mozilla::void_t from "ipc/IPCMessageUtils.h";
-
-namespace mozilla {
-namespace scache {
-
-struct EntryData {
-    nsCString key;
-    // This will be an empty array if data is present in the previous
-    // session's cache.
-    uint8_t[] data;
-};
-
-protocol PStartupCache
-{
-    manager PContent;
-
-parent:
-    async __delete__(EntryData[] entries);
-};
-
-} // namespace scache
-} // namespace mozilla